### PR TITLE
Add SDK Tracking

### PR DESCRIPTION
**Pull request by josephwegner**

### CODE REVIEW

File: keen/utilities.py [hunk 1]
```diff
@@ -0,0 +1,25 @@
+VERSION = None
+
+def version():
+  """
+  Retrives the current version of the SDK
```
Comment (masojus) [inline keen/utilities.py]: Nitpick: "Retrives" => "Retrieves"

Comment (dkador) [inline keen/utilities.py]: +1 - let's get that cleaned up

File: keen/utilities.py [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  """
+  Helper function to easily get the correct headers for an endpoint
+
+  :params api_key: The appropriate API key for the request being made
+  """
+
+  return {
+      "Content-Type": "application/json",
+      "Authorization": api_key,
+      "Keen-Sdk": "python-{0}".format(version())
```
Comment (dkador) [inline keen/utilities.py]: let's confirm capitalization. `Keen-SDK` vs `Keen-Sdk`. (i prefer the former but don't really care overall. let's just make sure we match! :) )

File: keen/utilities.py [hunk 3]
```diff
@@ -0,0 +1,28 @@
+import os
```
Comment (dkador) [inline keen/utilities.py]: nit: use 4 spaces for indentation to be consistent with other code.

File: keen/utilities.py (hunk 1 above)
Comment (josephwegner) [inline keen/utilities.py]: good eye!

File: keen/utilities.py (hunk 3 above)
Comment (josephwegner) [inline keen/utilities.py]: Resolved.

**Pull request merged**